libr/arch/p/arch_bf.c

```diff
@@ -134,24 +202,26 @@ static int assemble(RAsm *a, RAsmOp *op, const char *buf) {
 	r_strbuf_setbin (&op->buf, opbuf, n);
 	return n;
 }
+#endif
 
-RAsmPlugin r_asm_plugin_bf = {
+RArchPlugin r_arch_plugin_bf = {
 	.name = "bf",
-	.author = "pancake, nibble",
+	.author = "pancake",
 	.version = "4.0.0",
 	.arch = "bf",
 	.license = "LGPL3",
 	.bits = 16 | 32 | 64,
 	.endian = R_SYS_ENDIAN_NONE,
 	.desc = "Brainfuck",
-	.disassemble = &disassemble,
-	.assemble = &assemble
+	.decode = decode,
+//	.encode = encode
```

Comment:
remove

Reply:
no. this needs to be implemented, this is why its WIP